src/tests.rs

```diff
@@ -122,6 +122,12 @@ fn parse_empty_enum() {
     assert_debug_snapshot!(enum_type);
 }
 
+#[test]
+#[should_panic]
+fn reject_trailing_junk() {
+    dbg!(parse_declaration(quote::quote! {struct Good {} bad})).unwrap();
```

Comment:
This test is very busy, you could probably make it a bit more readable with [`Result::expect_err()`](https://doc.rust-lang.org/std/result/enum.Result.html#method.expect_err) and two lines:
```rs
#[test] // no should_panic
fn reject_trailing_tokens() {
    let tokens = quote::quote! { struct Good {} bad };
    parse_declaration(tokens).expect_err("trailing tokens should cause error");
}

Reply:
> `no should_panic`

Ehm…how? Anyway, I made the test look a bit more like the other tests. Maybe you like it better now? If not, the lowest effort variant is probably if you fix it yourself.